<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergerWindow</class>
 <widget class="QWidget" name="MergerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>710</height>
    </rect>
   </property>
   <widget class="QLabel" name="left_filename_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Первый фрагмент ПЗ</string>
    </property>
   </widget>
   <widget class="QLabel" name="right_filename_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Второй фрагмент ПЗ</string>
    </property>
   </widget>
   <widget class="QLabel" name="textout_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла с объединённым ПЗ в формате ЯПЗ</string>
    </property>
   </widget>
   <widget class="QLabel" name="xmlout_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла с объединённым ПЗ в формате XML</string>
    </property>
   </widget>
   <widget class="QLabel" name="innerout_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла с объединённым ПЗ на внутреннем формате</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="left_filename_value">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>first</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="right_filename_value">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>second</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textout_value">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>merged_rules</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xmlout_value">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>merged_xml</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="innerout_value">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>merged_inner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить фрагменты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_types_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Объединить типы</string>
    </property>
   </widget>
   <widget class="QLabel" name="log_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл с логом работы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="log_value">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>590</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>merger_log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить тестовыми данными</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_objects_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Объединить Объекты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить результат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_events_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Объединить События</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_intervals_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Объединить Интервалы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_rules_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Объединить Правила</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_rtr_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать РТР</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_dists_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать таблицу
 мер схожести</string>
    </property>
   </widget>
   <widget class="QLabel" name="rtr_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла с описанием РТР в формате csv</string>
    </property>
   </widget>
   <widget class="QLabel" name="dists_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла с описанием таблицы мер схожести</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rtr_value">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>extended_table</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dists_value">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>dists_table</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Главное окно</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>8</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
